### DISCUSSION TITLE
Is there any reason for the verified/known bot status anymore?

### DISCUSSION THREAD

**Discussion by ALazyMeme** [pajbot/pajbot#2339] [General]
Based on the [twitch docs](https://dev.twitch.tv/docs/irc/#rate-limits), known bots are basically completely useless and the only reason for verified bots is for increased message limits across multiple channels. We are still limited to sending 100/30s where the bot is a moderator. The whispers no longer work either due to the migration to helix-based whispers.

So is there any real reason to keep verified and known status' in pajbot?

**Comment by pajlada** [comment]
Nope, no reason for it for our current implementations
